privaterelay/utils.py

```diff
@@ -0,0 +1,37 @@
+from django.conf import settings
+
+def get_premium_countries_info_from_request(request):
+    country_code = _get_cc_from_request(request)
+    premium_countries = settings.PREMIUM_PLAN_COUNTRY_LANG_MAPPING.keys()
+    premium_available_in_country = country_code in premium_countries
+    return({
+        'country_code': country_code,
+        'premium_countries': premium_countries,
+        'premium_available_in_country': premium_available_in_country
+    })
+
+
+def _get_cc_from_request(request):
+    if 'X-Client-Region' in request.headers:
+        return request.headers['X-Client-Region'].lower()
+    if 'Accept-Language' in request.headers:
+        return get_premium_country_lang(request.headers['Accept-Language'])
```

Comment:
As I understand it `get_premium_country_lang` returns a tuple `(cc, lang)`. If we don't return just the country code, `get_premium_countries_info_from_request` will try to find the tuple as a whole in `premium_countries`, which it can't.

```suggestion
        return get_premium_country_lang(request.headers['Accept-Language'])[0]
```

Reply:
@groovecoder This change is blocking #1376. Could you review?

Reply:
Committed suggestion.